atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraExpiringKeyValueService.java

```diff
@@ -47,30 +48,24 @@
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
-public class CassandraExpiringKeyValueService extends CassandraKeyValueServiceImpl implements ExpiringKeyValueService {
-
+public final class CassandraExpiringKeyValueService extends CassandraKeyValueServiceImpl
```

Comment:
I see what you did here. We should probably move this class internally. Filed https://github.com/palantir/atlasdb/issues/2309 to track.